3Edges/3edges-deployments
# Terraform IaC for 3edges on AWS

SUMMARY
This PR is to merge to the dev branch

TEST PLAN
* Clone the repo 
* Follow the README file & customize the values in terraform.tfvars
* Run the terraform (run.sh) script
* Connect to the cluster, verify the cluster health and access the 3edges platform
